tools/codegen/gen.py

```diff
@@ -803,7 +792,7 @@ def compute_declaration_yaml(f: NativeFunction) -> object:
         ('device_guard', f.device_guard),
         ('with_gil', False),
         ('deprecated', False),
-        ('has_math_kernel', f.dispatch is not None and 'Math' in f.dispatch),
+        ('has_math_kernel', 'Math' in f.dispatch),
```

Comment:
This field is only used for the old RegistrationDeclarations.h. Should be good to remove now. ;)